Makefile

```diff
@@ -0,0 +1,29 @@
+# HTML files that are generated from Markdown sources.
+HTML_FROM_MD_TARGETS=README.html README-with-toc.html
+
+.PHONY: all
+all: spec.html $(HTML_FROM_MD_TARGETS)
+
+.PHONY: clean
+clean:
+	-rm spec.html
+	-rm $(HTML_FROM_MD_TARGETS)
+
+# Updates Bikeshed's datafiles. Run this regularly to ensure you're not linking
```

Comment:
This might not be necessary, bikeshed should update automatically ([ref](https://speced.github.io/bikeshed/#:~:text=By%20default%2C%20Bikeshed%20checks%20how,than%20a%20few%20days%20old)), so might want to update the comment

Reply:
Oh weird, I actually added this because I hit a scenario where my local build succeeded but CI failed. I suspect it's due to the  "more than a few days old" part.

I can sharpen the comment a bit:

```
# Updates Bikeshed's datafiles. Bikeshed takes an eventual-consistency approach
# to updating its datafiles. Consequently, local builds may incorrectly succeed
# when referenced documents were recently updated.
```

WDYT?

Reply:
Maybe let's keep a reference to the time frame, e.g. something like:
Updates Bikeshed's datafiles. Bikeshed automatically updates them if they're more than a few days old, but this will ensure you have the latest version.

Reply:
That's a little clearer, thanks.